test-apps/display-test-app/src/common/DtaRpcInterface.ts

```diff
@@ -15,8 +17,13 @@ export class DtaRpcInterface extends RpcInterface {
   /** The types that can be marshaled by the interface. */
   public static types = () => [];
 
+  /** The backend server, when running on a browser */
```

Comment:
What is it when not running on a browser? If it can be `undefined`, express that.

Reply:
Done